index.html

```diff
@@ -5289,7 +5289,8 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 			<rdef>menu</rdef>
 			<div class="role-description">
 				<p>A type of <a>widget</a> that offers a list of choices to the user.</p>
-				<p>A menu is often a list of common actions or functions that the user can invoke. The <code>menu</code> <a>role</a> is appropriate when a list of menu items is presented in a manner similar to a menu on a desktop application.</p>
+				<p>A menu is a container, generally visually rendered as a popup or overlay, for a set of menu items that can be invoked to perform an action or function. The function is almost always directly related to the element, or a close relation to the element that the user activated to invoke the menu. Activating a menu item both performs the function, and results in the automatic dismissal of the menu.</p>
+				<p>The <code>menu</code> <a>role</a> is appropriate when a set of menu items is presented in a manner similar to a popup menu for a desktop application. For instance, a menu could be used to represent a context menu for its invoking element, or it would be used to render sub-menus for items of a <rref>menubar</rref> or another <code>menu</code> popup.</p>
```

Comment:
```suggestion
				<p>The <code>menu</code> <a>role</a> is appropriate when a set of menu items is presented in a manner similar to a popup menu. For instance, a menu could be used to represent a context menu for its invoking element, or it would be used to render sub-menus for items of a <rref>menubar</rref> or another <code>menu</code> popup.</p>
```